### DISCUSSION TITLE
How do I autoscroll using middle click?

### DISCUSSION THREAD

**Discussion by rtahm** [artginzburg/MiddleClick#72] [Q&A] [answered]
I would like to middle click and use the mouse to auto scroll by having my mousing above from the origin point where I clicked. The further I am away from the origin of the click, I would like to have higher the auto scrolling speed. 

Thanks

**Comment by artginzburg** [comment, marked as answer]
You very much can just do that in Firefox. Probably in Chrome too via an extension.
In Firefox, just middle-click once on a page, and move the cursor towards the intended direction

> Sorry for necroposting, but I hope someone finds it helpful.
